### ISSUE TITLE
Group layer

### ISSUE THREAD

**Issue by clabe45** [clabe45/vidar#65]
The group layer has zero or more child layers. It composites all visual children onto its canvas and mixes all audio children into its audio node.

**See also:**
- `vd.layer.AudioMixin` (not implemented)
- `vd.layer.Visual`
- `vd.Movie#_render` (to see how to composite visual layers)

**Comment by audreyspency** [comment]
did u solve that issue ?

**Comment by clabe45** [comment]
Nope, would you like me to assign you to it?

**Comment by codeSourc3** [comment]
Would it be a good idea to split the responsibilities of grouping audio and visual children into two separate classes? We could even have them inherit from a base class.

**Comment by clabe45** [comment]
Good idea. We also need an audio + video group layer though (for `<video>` elements). The best option I see is to make an audio group mixin, a visual group mixin and then a class that uses the audio group mixin, a class that uses the visual group mixin, and a class that uses both.

We are already doing this with the `Audio` and `Visual` layers (but I need to make another issue to make a mixin for `Visual`, it's kind of inconsistent right now).

**Comment by clabe45** [comment]
Just updated the description, what do you think @codeSourc3?

**Comment by codeSourc3** [comment]
Sounds good

**Comment by codeSourc3** [comment]
Sounds good
